include/antler/project/location.hpp

```diff
@@ -22,12 +25,18 @@ namespace antler::project::location {
 /// @note requires `gh` is installed and user is authenticated or the repo is public.
 /// @param l Location to evaluate.
 /// @return true if l is a repo, calls `gh` to test.
-[[nodiscard]] bool is_github_org_repo_shorthand(std::string_view l);
+[[nodiscard]] bool is_github_shorthand(std::string_view l);
 /// @param l  Location to evaluate.
 /// @return true if l is reachable.
 bool is_reachable(std::string_view l);
 /// @param l  Location to evaluate.
 /// @return true if l looks like a url.
 bool is_url(std::string_view l);
 
+bool clone_github_repo(const std::string& org, const std::string& repo, const std::string& branch, uint32_t jobs, system::fs::path dest = ".");
+bool clone_git_repo(const std::string& url, const std::string& branch, uint32_t jobs, system::fs::path dest = ".");
```

Comment:
I'm thinking `jobs` should become the final parameter and default to `0`  OR maybe it's simpler and better to honor the users `submodule.fetchJobs` setting.

Reply:
#28 written to address unfriendly jobs behavior.